index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Portfolio Project</title>
+  </head>
+  <body>
+    <!-- Header Section -->
+    <header>
+      <a class="logo-text" href="#">Welcome</a>
+      <div class="hamburger">
+        <span class="bar"></span>
+        <span class="bar"></span>
+        <span class="bar"></span>
+      </div>
```

Comment:
- [ ] [Optional] I would suggest you please replace the div tag with the nav tag for the hamburger button for a better semantics reason.

Reply:
Thank you @roshan-bajgain , I will replace the tags now. Thank you for your feedback